ci-operator/config/openshift-assisted/assisted-installer-ui/openshift-assisted-assisted-installer-ui-master.yaml

```diff
@@ -0,0 +1,42 @@
+base_images:
+  node18:
+    name: nodejs-18-minimal
+    namespace: edge-infrastructure
+    tag: latest
+  openshift_release_golang-1.21:
+    name: release
+    namespace: openshift
+    tag: golang-1.21
+  openshift_ubi-micro_9:
+    name: ubi-micro
+    namespace: ocp
+    tag: "9"
+build_root:
+  project_image:
+    dockerfile_literal: |
+      FROM quay.io/centos/centos:stream9
```

Comment:
Any specific reason for using centos?

Reply:
no, I just saw it being used in the first project in `openshift-assisted`  folder https://github.com/openshift/release/blob/master/ci-operator/config/openshift-assisted/assisted-events-scrape/openshift-assisted-assisted-events-scrape-master.yaml#L10 
so I used the same.

Should I use different one ? Any suggestion ?